openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhooksV1.tsx

```diff
@@ -0,0 +1,183 @@
+  const [filteredData, setFilteredData] = useState<Array<Webhook>>(data);
+
+  const getFilteredWebhooks = () => {
+    return selectedStatus.length
+      ? data.filter(
+          (item) => item.status && selectedStatus.includes(item.status)
+        )
+      : data;
+  };
+
+  const fetchRightPanel = () => {
+    return (
+      <Card data-testid="data-summary-container" style={leftPanelAntCardStyle}>
+        <div className="tw-my-2">
+          The webhook allows external services to be notified of the metadata
+          change events happening in your organization through APIs. Register
+          callback URLs with webhook integration to receive metadata event
+          notifications. You can add, list, update, and delete webhooks.
+        </div>
+      </Card>
+    );
+  };
+
+  const fetchErrorPlaceHolder = (message: string) => {
+    return (
+      <ErrorPlaceHolder>
+        <p className="tw-text-center">{message}</p>
+        <p className="tw-text-center">
+          <NonAdminAction position="bottom" title={TITLE_FOR_NON_ADMIN_ACTION}>
+            <Button
+              className={classNames('tw-h-8 tw-rounded tw-my-3', {
+                'tw-opacity-40': !isAdminUser && !isAuthDisabled,
+              })}
+              data-testid="add-webhook-button"
+              size="small"
+              theme="primary"
+              variant="contained"
+              onClick={onAddWebhook}>
+              Add Webhook
+            </Button>
+          </NonAdminAction>
+        </p>
+      </ErrorPlaceHolder>
+    );
+  };
+
+  useEffect(() => {
+    setFilteredData(getFilteredWebhooks());
+  }, [data, selectedStatus]);
+
+  if (data.length === 0) {
+    return fetchErrorPlaceHolder('No webhooks found');
+  }
+
+  return (
+    <div className="tw-flex tw-justify-between tw-gap-4">
+      <div className="tw-w-full">
+        <div className="tw-flex tw-items-center tw-justify-between">
+          <Select
+            bordered={false}
+            className="tw-min-w-64 tw-text-body webhook-filter-select"
+            mode="multiple"
+            options={statuses}
+            placeholder="Filter by status"
+            onChange={onStatusFilter}
+          />
+          {filteredData.length > 0 && (
+            <NonAdminAction
+              position="bottom"
+              title={TITLE_FOR_NON_ADMIN_ACTION}>
+              <Button
+                className={classNames('tw-h-8 tw-rounded tw-mb-3', {
+                  'tw-opacity-40': !isAdminUser && !isAuthDisabled,
+                })}
+                data-testid="add-webhook-button"
+                size="small"
+                theme="primary"
+                variant="contained"
+                onClick={onAddWebhook}>
+                Add Webhook
+              </Button>
+            </NonAdminAction>
+          )}
+        </div>
+        {filteredData.length ? (
+          <>
+            {filteredData.map((webhook, index) => (
```

Comment:
We can club this as both check for same conditions
line no: 133 & 151

Reply:
no, we can't merge this as it's in two different div